scripts/generate_dump

```diff
@@ -235,14 +250,15 @@ save_cmd_all_ns() {
 #  None
 ###############################################################################
 copy_from_docker() {
+    trap 'handle_error $? $LINENO' ERR
     local start_t=$(date +%s%3N)
     local end_t=0
     local docker=$1
     local filename=$2
     local dstpath=$3
     local timeout_cmd="timeout --foreground ${TIMEOUT_MIN}m"
 
-    local touch_cmd="sudo docker exec -i ${docker} touch ${filename}"
+    local touch_cmd="sudo docker exec -t ${docker} touch ${filename}"
```

Comment:
>-t [](http://example.com/codeflow?start=38&length=2)

No need to add `-t`. In some use case, there is no tty.

Reply:
Okay, will remove

Reply:
Handled